### ISSUE TITLE
Possible adjustment to Slide 23 - improved clarity.

### ISSUE THREAD

**Issue by shanecoughlan** [OpenChain-Project/curriculum#9]
From Dirk:

==

Concerning the  OpenChain Curriculum (v1-1)
Page 23  "Check Your Understanding (of Chapter 2)"  =>  Notices 

Maybe I am wrong but I just wondered, if

"License reciprocity means that the derivative work of the copyrighted work
must be made available under the same license."

should be changed for clarification to 

"License reciprocity means that in the case of redistribution the derivative
work of the copyrighted work must be made available under the same license."

or like this:
"License reciprocity means that in the case that you decide to redistribute
your derivative work of the copyrighted work, such a redistribution must be
made available under the same license."

because you are not obliged to redistribute your derivative work. You just
can use it for your own or internally in your company for companies
purposes.

Best regards

Dirk

---------------------------------------------------------------------------
Karlsruhe Institute für Technology (KIT)
INNOVATION AND RELATIONS MANAGEMENT (IRM)
Intellectual Property Management

Dr. Dirk Feuchter

**Comment by andrewjskatz** [comment]
Agreed - Dirk is correct. I'd suggest a hybrid approach retaining the original wording by combining it with Dirk's suggestion:

"License reciprocity means that if you redistribute the copyright work, it, and any derivative of it, must be made available under the same license terms.

[I can't remember if we're adopting US or British spelling conventions, but I'm assuming US]
